staging/src/k8s.io/apiserver/pkg/audit/policy/checker_test.go

```diff
@@ -88,6 +100,11 @@ func TestChecker(t *testing.T) {
 			Verbs:     []string{"get"},
 			Resources: []audit.GroupResources{{Resources: []string{"pods"}}},
 		},
+		"getPodLogs": {
+			Level:     audit.LevelRequest,
+			Verbs:     []string{"get"},
+			Resources: []audit.GroupResources{{Resources: []string{"pods/logs"}}},
```

Comment:
This break one unit test because "pods/logs" doesn't not consistent with line 71.